### ISSUE TITLE
Overview of albums does not show the artist name, but 'albu'

### ISSUE THREAD

**Issue by EelcoA** [janbar/noson-app#12]
Both my 7Digital as my Google Play album overview show 'albu' where on my local (NAS) album overview the artist name is shown.

This is Google Play:
![afbeelding](https://user-images.githubusercontent.com/9035993/28491916-7a6e3ce8-6ef9-11e7-8f55-1b6056744021.png)

This is my NAS:
![afbeelding](https://user-images.githubusercontent.com/9035993/28491928-9caf37f8-6ef9-11e7-84e6-67c2f2b7f4e2.png)

**Comment by janbar** [comment]
Yes, that is little weird comparing with local music library. Until now all services I tested provide the attribute 'artist' instead 'author' or 'composer'. I fixed it for items type album and song. So I updated the preferred order of metadata to display here, as the following:
1 - description
2 - author
3 - composer
4 - artist
5 - else the type of item: Album, Song, Playlist etc...

**Comment by janbar** [comment]
Included in version 2.4.5

**Comment by janbar** [comment]
I added the case 4 explained above. So now, it will show first the 'description' if exists, else the author, else the composer, else the artist (tested services provide this one), else the item type.
Please let know if it is fixed with the latest version for you. Thanks.

**Comment by EelcoA** [comment]
This one works, with 7Digital at least, can't check Google Play. See my comment on the other issue.
Thanks again :-)
